bvm/ballerina-profiler/src/main/java/io/ballerina/runtime/profiler/Profiler.java

```diff
@@ -159,6 +162,14 @@ private void handleProfilerArguments(String[] args) {
                     this.targetDir = args[i + 1];
                     addToUsedArgs(args, usedArgs, i);
                 }
+                case "--sourceroot" -> {
+                    this.sourceRoot = args[i + 1];
+                    addToUsedArgs(args, usedArgs, i);
+                }
+                case "--profilerDebug" -> {
```

Comment:
Normally for command line option we are using multiple words seperated by dashes.
ex. --profiler-debug or --debug-profiler

Reply:
We can fix this as separate PR.